background/get-addon-settings.js

```diff
@@ -297,6 +297,45 @@ chrome.storage.sync.get([...ADDON_SETTINGS_KEYS, "addonsEnabled"], (storageItems
               madeAnyChanges = madeChangesToAddon = true;
             }
           );
+
+          updatePresetIfMatching(
+            settings,
+            6,
+            // "Scratch default colors (blue)" preset
+            {
+              page: "#fcfcfc",
+              box: "#ffffff",
+              gray: "#f2f2f2",
+              blue: "#e9f1fc",
+              input: "#fafafa",
+              link: "#4d97ff",
+              footer: "#f2f2f2",
+              border: "#0000001a",
+            },
+            () => {
+              settings.messageIndicatorOnMessagesPage = "#ffab1a";
+              madeAnyChanges = madeChangesToAddon = true;
+            }
+          );
+          updatePresetIfMatching(
+            settings,
+            7,
+            // "Experimental Dark (blue)" preset
+            {
+              page: "#202020",
+              box: "#282828",
+              gray: "#333333",
+              blue: "#252c37",
+              input: "#202020",
+              link: "#4d97ff",
+              footer: "#333333",
+              border: "#606060",
+            },
```

Comment:
All these settings are exactly the same as migration #3, confirmed through Visual Studio Code search.